include/onnxruntime/core/session/onnxruntime_c_api.h

```diff
@@ -513,6 +513,71 @@ ORT_API_STATUS(OrtGetValueCount, const OrtValue* value, size_t* out);
 ORT_API_STATUS(OrtCreateValue, OrtValue** const in, int num_values, enum ONNXType value_type,
                OrtValue** out);
 
+/*
+ * EXPERIMENTAL APIS - Subject to change. Released as a preview to get feedback and enable early testing
+*/
+
+/*
+ * Steps to use a custom op:
+ *   1 Create an OrtCustomOpDomain with the domain name used by the custom ops
+ *   2 Create an OrtCustomOp structure for each op and add them to the domain
+ *   3 Call OrtAddCustomOpDomain to add the custom domain of ops to the session options
```

Comment:
The 3rd step is not needed. The custom op domain should be added when adding custom op (with custom domain) in inference session level.